File: 27-remove-element/remove-element.cs
public class Solution {
    public int RemoveElement(int[] nums, int val) {
        int n = nums.Length;
        int i = 0, j=0;

        while (j<n) {
            if (nums[j]!=val) {
                nums[i] = nums[j];
                i += 1;
            }
            j += 1;
        }

        return i;
    }
}

File: 27-remove-element/remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        cur = 0
        n = len(nums)
        for i in range(n):
            if nums[i]!=val:
                nums[cur] = nums[i]
                cur += 1
        return cur
        